### PR TITLE
fix for min tls version to v1.2

### LINKED ISSUE #2118: Fix: min tls setting for webhook server
### What happened?

default TLS version for webhook server on port 9443 is enabled with TLSv1.0 and TLSv1.1, which are deprecated due to known vulnerabilities and should be avoided.

this can be refer with issues from https://github.com/open-telemetry/opentelemetry-operator/issues/1225

### Steps to reproduce

running sslscan on port 9443

### Expected behavior

TLSv1.0 and TLSv1.1 should be disable

### Relevant log output

_No response_

### Screenshot

image
this is my local test, where i portforward port 9443 to 10001

### Additional context

_No response_

### Jaeger backend version

_No response_

### SDK

_No response_

### Pipeline

_No response_

### Stogage backend

_No response_

### Operating system

_No response_

### Deployment model

_No response_

### Deployment configs

_No response_

### PR DESCRIPTION
**Pull request by kangsheng89**
## Which problem is this PR solving?
Fix #2118

## Short description of the changes
Adding min tls version for webhook server during bootstraping

### CODE REVIEW

**Comment by kangsheng89** [comment]
after fix on my local, with portforward to 10001, clearly view that tyhe webhook server port 9443 is enabled with min TLSv1.2
image

File: pkg/cmd/start/bootstrap.go [hunk 1]
```diff
@@ -52,6 +53,10 @@ import (
 	esv1 "github.com/openshift/elasticsearch-operator/apis/logging/v1"
 )
 
+// We should avoid that users unknowingly use a vulnerable TLS version.
+// The defaults should be a safe configuration.
+const defaultMinTLSVersion = tls.VersionTLS12
```
Comment (frzifus) [inline pkg/cmd/start/bootstrap.go:58]: Would it make sense to introduce a way to change the minTLS version?

Comment (kangsheng89) [reply pkg/cmd/start/bootstrap.go:58]: Yes definitely, may b we can do it by adding a configuration from cli argument? May b u can point me on this?

**Comment by frzifus** [comment]
wdyt @rubenvp8510 ?

File: pkg/cmd/start/bootstrap.go (hunk 1 above)
Comment (frzifus) [reply pkg/cmd/start/bootstrap.go]: seems you already figured it out :)

**Comment by kangsheng89** [comment]
@frzifus  i will update for TLS cipher as well. Hehe pending..LOL

**Comment by rubenvp8510** [comment]
This looks good to me!

**Review by rubenvp8510 — APPROVED** [review]

**Comment by kangsheng89** [comment]
currentlt the CICD is failing 
+---------------------------------------------------------+--------+
|                          NAME                           | RESULT |
+---------------------------------------------------------+--------+
| examples/examples_simple_prod                           | passed |
| examples/examples_with_sampling                         | passed |
| examples/examples_with_cassandra                        | passed |
| examples/examples_with_badger_and_volume                | passed |
| examples/examples_with_badger                           | passed |
| examples/examples_simplest                              | passed |
| examples/examples_simple_prod_with_volumes              | passed |
| examples/examples_auto_provision_kafka                  | passed |
| examples/examples_service_types                         | passed |
| examples/examples_collector_with_priority_class         | passed |
| examples/examples_business_application_injected_sidecar | passed |
| examples/examples_agent_with_priority_class             | passed |
| examples/examples_all_in_one_with_options               | failed |
| examples/examples_agent_as_daemonset                    | passed |
+---------------------------------------------------------+--------+

@frzifus  can u point me on how i can fix this?

**Comment by kangsheng89** [comment]
@frzifus can help to rerun the CICD pipeline?

**Comment by frzifus** [comment]
@kangsheng89 it did already pass :)

**Pull request merged**